languages/java/config.json

```diff
@@ -9,6 +9,12 @@
   },
   "exercises": {
     "concept": [
+      {
+        "slug": "basics",
```

Comment:
* the slug is `arrays`
* uuid should be unique, you can create it as you want. If you need an advice, I use `configlet` to create it (see [here](https://github.com/exercism/configlet#uuid) for more details).
* prerequisites is the array containing the slugs of concepts prerequisites  to solve this exercise.

So the result should be something as
```
{
        "slug": "arrays",
        "uuid": "XXXXXXXXX",
        "concepts": ["arrays"],
        "prerequisites": ["basics", "booleans"]
      }
```